packages/tools/src/tools/segmentation/strategies/utils/initializeTracking.ts

```diff
@@ -0,0 +1,27 @@
+import type { InitializedOperationData } from '../BrushStrategy';
+
+/**
+ * Sets up tracking for use by preview and other services.
+ * The tracking is either the preview object, or the previewVoxelValue.
+ * Re-using this existing preview means that the same segment colour and already
+ * tracked values can be extended.
+ */
+export default {
```

Comment:
let's rename this file to something else since we have tracking in OHIF for measurements and it would be confusing. Another question is that why you can't merge this one into the `createInitialized` of the `initializePreview` ?